IdentityCore/src/network/request/MSIDAADBrowserPRTResponse.h

```diff
@@ -0,0 +1,22 @@
+//
+//  MSIDAADBrowserPRTResponse.h
+//  IdentityCore iOS
+//
+//  Created by Rohit Narula on 1/13/20.
+//  Copyright © 2020 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSIDAADBrowserPRTResponse : NSObject
+
+@property (nonatomic, nullable, readonly) NSHTTPURLResponse *response;
```

Comment:
Looks like a representation of any NSURLSession response. Should we call it something more generic?

Reply:
Agree. Renamed to MSIDUrlResponse . Also, as discussed removed PRTResponseSerializer and instead made MSIDRequestSerialization generic to accept headers along with parameters.